packages/vulcan-lib/lib/server/apollo-server/apollo_server.js

```diff
@@ -153,7 +153,10 @@ export const onStart = () => {
   const apolloServerOptions = {
     engine: engineConfig,
     schema: GraphQLSchema.executableSchema,
-    formatError,
+    formatError: error => {
+      getServerLogger().info(error.message, error);
```

Comment:
should be calling `.error()` on the logger here instead of `.info()`? might not matter much